ui/package.json

```diff
@@ -86,10 +84,8 @@
     "eslint-plugin-vue": "^7.0.0",
     "less": "^3.0.4",
     "less-loader": "^5.0.0",
-    "node-sass": "^4.0.0",
     "sass": "^1.49.9",
     "sass-loader": "^8.0.2",
-    "uglifyjs-webpack-plugin": "^2.2.0",
```

Comment:
Don't we want to minimise/compress the UI build?

Reply:
My apologies, I tried to delete it to check the audit fix but made a mistake. Will add it back in.